dataland-backend/src/test/kotlin/org/dataland/datalandbackend/controller/DataControllerTest.kt

```diff
@@ -36,10 +36,10 @@ internal class DataControllerTest(
 ) {
     val testDataProvider = TestDataProvider(objectMapper)
 
-    val testDataType = DataType.valueOf("sme")
+    val testDataType = DataType.valueOf("eutaxonomy-non-financials")
     val storedCompanyEntity = testDataProvider.getEmptyStoredCompanyEntity()
-    val someSmeData = testDataProvider.getEmptySmeDataset()
```

Comment:
Why those changes? Did this no longer work with sme data?

Reply:
There was a required field added to SME and this method just returned an empty object. However, the type of dataset does not really matter here, thats why it was rather changed in data type.